[DOCS] Updates to how to get support (#9809)
great-expectations/great_expectations: docs/docusaurus/versioned_docs/version-0.18/resources/get_support.md

description: Learn what resources are available to help you resolve issues with GX Cloud and GX OSS.
---

> ## Where to get support

Reviewer: ![image](https://github.com/great-expectations/great_expectations/assets/104205257/2f6be469-2299-479e-847f-c5c488f22e1d)

Add one or two sentences that describe the content that follows the heading and its purpose. Alternatively, remove the existing H2 heading and promote the H3 headings. For help with heading structure, see [Heading and title format](https://developers.google.com/style/headings#heading-and-title-format).

### GX Cloud support

> GX Cloud customers can reach the GX support team through the following channels.

Reviewer: ```suggestion
Use one of the following methods to request support:
```
You don't need _GX Cloud_ in the introductory statement because the heading already contains this information. Users will understand that you're describing the options for requesting GX Cloud support without the need to repeat the product name.

> - Sending an email to support@greatexpectations.io

Reviewer: ```suggestion
- Send an email to support@greatexpectations.io.
```
If you're adding punctuation to lists, add it consistently.

> - Creating a new topic on the [GX Cloud Discourse forum](https://discourse.greatexpectations.io/c/cloud-support/17).

Reviewer: ```suggestion
- Create a new topic on the [GX Cloud Discourse forum](https://discourse.greatexpectations.io/c/cloud-support/17).
```

> - Reaching out in the [GX Cloud Slack community channel](https://greatexpectationstalk.slack.com/archives/C051D941XAL).

Reviewer: ```suggestion
- Post a message in the [GX Cloud Slack community channel](https://greatexpectationstalk.slack.com/archives/C051D941XAL).
```

### GX OSS support

> If you're using GX OSS only, you can get support via the community.

Reviewer: ```suggestion
Use one of the following methods to request support:
- Post a message in the [GX Slack community channel](https://greatexpectationstalk.slack.com/archives/CUTCNHN82). 
- Open an issue in the [GX GitHub repository](https://github.com/great-expectations/great_expectations) when you encounter a bug.
```
![image](https://github.com/great-expectations/great_expectations/assets/104205257/bfa7882e-9f67-4dbb-855b-2ed8b86a4c3b)

> - Chat with community members about general issues and seek solutions on the [GX Slack community channel](https://greatexpectationstalk.slack.com/archives/CUTCNHN82).

Reviewer: ```suggestion

```

> - If you've found a bug related to OSS, open an issue in our [GX GitHub repository](https://github.com/great-expectations/great_expectations).

Reviewer: ```suggestion

```

> ## How to effectively get support

Reviewer: ![image](https://github.com/great-expectations/great_expectations/assets/104205257/8c7cad7f-29de-4c3b-8d15-b3441347225c)

Add one or two sentences that describe the content that follows the heading and its purpose. Alternatively, remove the existing H2 heading and promote the H3 headings. For help with heading structure, see [Heading and title format](https://developers.google.com/style/headings#heading-and-title-format).
